code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -155,6 +155,7 @@
 /obj/structure/cult/functional/altar/Initialize(mapload)
 	. = ..()
 	icon_state = SSticker.cultdat?.altar_icon_state
+	cooldowntime = world.time + 60 SECONDS
```

Comment:
I mean like with a define so no magic numbers as well, remember to undefine it later